R/RLearner_regr_gbm.R

```diff
@@ -30,10 +32,39 @@ trainLearner.regr.gbm = function(.learner, .task, .subset, .weights = NULL, ...)
   f = getTaskFormula(.task)
   if (is.null(.weights)) {
     f = getTaskFormula(.task)
-    gbm::gbm(f, data = getTaskData(.task, .subset), ...)
+
+    params <- list(...)
+    if("alpha" %in% names(params)){
+      alpha <- params$alpha
+      params$alpha <- NULL
+    } else {
+      alpha <- 0.5
+    }
+    if(params$distribution %in% "quantile"){
+      params$distribution <- list(name = "quantile", alpha = alpha)
+    }
+    params$formula <- f
+    params$data <- getTaskData(.task, .subset)
+
+    do.call(gbm::gbm, params)
```

Comment:
Why is this extra code needed here?

Reply:
``gbm`` use two ways to pass ``distribution`` parameters : 

- simple character as "gaussian" or "laplace"
- named list, for "quantile" like ``list(name = "quantile", alpha = 0.5)

So we have to control the distribution and format well ``distrbution`` argument

Reply:
ok.